lib/ts/recipe/emailpassword/api/signin.ts

```diff
@@ -16,10 +16,11 @@
 import { send200Response } from "../../../utils";
 import { validateFormFieldsOrThrowError } from "./utils";
 import { APIInterface, APIOptions } from "../";
+import { debugLoggerWithCode, infoLoggerWithCode, loggerCodes } from "../../../logger";
 
 export default async function signInAPI(apiImplementation: APIInterface, options: APIOptions): Promise<boolean> {
     // Logic as per https://github.com/supertokens/supertokens-node/issues/20#issuecomment-710346362
-
+    infoLoggerWithCode[loggerCodes.API_CALLED]("SignInAPI");
```

Comment:
This call signature is fairly ugly; I think it'd be better if we exported these log functions as named exports and used them like `logAPICalled("...")` instead. This has the added benefit that if we later want to tweak the level of a message type, we'd only have to update it in one place. I think the API implementations should avoid knowing/caring about log levels.

Reply:
Agreed